.changeset/neat-adults-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/scaffolder-backend-module-utils': minor
```

Comment:
No, the original `backend.add(import('@roadiehq/scaffolder-backend-module-utils/new-backend'));` will still continue to work until the removal of [new-backend.ts](https://github.com/RoadieHQ/roadie-backstage-plugins/blob/main/plugins/scaffolder-actions/scaffolder-backend-module-utils/src/new-backend.ts). So, I didn't consider it a major change

Reply:
I was thinking of the v1 plugin import, like this:

```
import plugin from '@roadiehq/scaffolder-backend-module-utils';

plugin.createSleepAction
```

Reply:
Ah, are you talking about this [change](https://github.com/RoadieHQ/roadie-backstage-plugins/pull/1711/files#diff-0c4e157e6d32a7594128096c196742f82d5b2ca7b1eb0dc0acdd00443d49ec3fR16)? I forgot that I had combined all of the actions. Testing locally in this repo all of the actions continue to show up using the old backend system as well.

Reply:
How are you importing them.. I hope I am mistaken.

I think there is a difference between something like this:

```
import plugin from '@roadiehq/scaffolder-backend-module-utils';
plugin.createSleepAction
```

and this:

```
import { createSleepAction } from '@roadiehq/scaffolder-backend-module-utils';
```

Reply:
Sorry I could be mistaken, of course.

Reply:
Are you doubly sure I am wrong.. I am kind of feeling like I could be wrong.. but then if I am right, it could be worth a major release?

Do please tell me if I am being too pedantic, or still missing something. I don't want to come across too forceful, because I am delighted that you are making this pull request.

I just checked and this seems to work pre this change:

```
import utils from '@roadiehq/scaffolder-backend-module-utils';

...
export const createActions = (options: {
  ...
}): TemplateAction<any>[] => {
  const {
    ...
  } = options;
  return [
    utils.createParseFileAction(),
    utils.createSerializeYamlAction(),
    utils.createJSONataAction(),
    utils.createSerializeJsonAction(),
    utils.createYamlJSONataTransformAction(),
    utils.createJsonJSONataTransformAction(),
    utils.createReplaceInFileAction(),
    utils.createSleepAction({ maxSleep: 180 }),
....
```

After this change, I guess this would stop working?

Reply:
No, you are not too pedantic. Nothing wrong with scrutiny in my eyes, especially considering that I am not even a part of this project.

And yep, looks like I did indeed miss a scenario. `import utils from '@roadiehq/scaffolder-backend-module-utils';` works previous and now no longer does. It would have to be imported like the following `import * as utils from '@roadiehq/scaffolder-backend-module-utils';`

If you are still ok with the changes included in this PR, then I am completely fine with upping this to a major release.

Reply:
@punkle, I went ahead and bumped it to a major, just to be on the safe side.